packages/gatsby-plugin-emotion/src/gatsby-node.js

```diff
@@ -1,27 +1,7 @@
 export const onCreateBabelConfig = ({ actions }, pluginOptions) => {
-  const pragmaName = `___EmotionJSX`
-
-  actions.setBabelPlugin({
-    name: `babel-plugin-jsx-pragmatic`,
+  actions.setBabelPreset({
```

Comment:
This should have probably been what we did from the beginning.

Doing some cursory reading--this doesn't require us to set the `jsx` pragma, correct? I'm not sure I like that approach, if so.

Reply:
(although this wasn't out yet--so heh, can't really fault us for that)

Reply:
heh yeah, the nice "single preset" approach wasn't out yet when the original emotion changes were made to gatsby plugin related stuff. It was still the piecemeal approach. 

Making this change *does not* require anyone to manually use the jsx pragma. The pragma is applied universally automatically. You can see [in the preset](https://github.com/emotion-js/emotion/blob/b966abd80cc4817cd27f2e4e6c7994cb2faf9043/packages/babel-preset-css-prop/src/index.js) that it's basically the same as what we were doing before by providing `jsx` and setting the pragma (with the React.Fragment settings, etc) and the core emotion plugin.